checklists/network_appdelivery_checklist.en.json

```diff
@@ -232,6 +232,7 @@
             "guid": "43fae595-8a32-4299-a69e-0f32c454dcc9",
             "id": "A01.41",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.network/applicationgatewaywebapplicationfirewallpolicies" | where properties['customRules'][0]['ruleType'] == "RateLimitRule" | distinct id, name",
```

Comment:
Please use single quotes for strings inside of the graph field. Additionally, you want to use the case-insensitive operator `=~` rather than `==`.

Thanks!